### ISSUE TITLE
Proper validation/ message display on respective fields (Admin Login)

### ISSUE THREAD

**Issue by deleenananaa** [Tenzing07/One-Byte-Food#8]
To enhance user experience and accessibility, validations and error handling for admin login should be displayed in a popup or dialogue box within the system. The bug is shown in the screenshot below:
![image](https://github.com/Tenzing07/One-Byte-Food/assets/119421004/0b0181cb-e30a-4191-abaf-7b214b7689b4)

**Comment by Tenzing07** [comment]
i have added the popup within the dialog box if the login credentials are incorrect
